app/main/posts/views/filters/active-search-filters.directive.js

```diff
@@ -39,7 +39,7 @@ function ActiveSearchFilters($translate, $filter, PostFilters, _, FilterTransfor
             var currentFilters = angular.copy(PostFilters.getUIActiveFilters(filters));
             FilterTransformers.rawFilters = angular.copy(filters);
             // Remove set filter as it is only relevant to collections and should be immutable in that view
-            delete currentFilters.set;
+            // delete currentFilters.set;
```

Comment:
can we remove it + the comment?   (I know you didn't change it, I can remove it in my PR,, just asking if it'd be ok  )

Reply:
I changed it... I have no idea what impact that had. If its ok to drop that then yes remove the line